src/tools/isabelle_executor.py

```diff
@@ -107,10 +107,11 @@ class IsabelleExecutor:
     assms_regex = r"assms_(\d+):"
     assms_regex_match = re.compile(assms_regex)
 
-    # Proof automation tactics
-    auto_tactics = ["simp", "auto", "blast", "metis", "argo", "linarith", "presburger", "algebra", "fast", "fastforce", "force", "meson", "satx"]
+    # Proof automation tactics: [tactics from LYRA] + [tactics from the *try0* keyword]
+    auto_tactics = ["auto", "simp", "blast", "fastforce", "force", "eval", "presburger", "sos", "arith", "linarith", "(auto simp: field_simps)",
+                    "metis", "argo", "algebra", "fast", "meson", "satx"]
```

Comment:
first row is tactics from LYRA
second row is extra tactics from `try0`, which is an Isabelle keyword that similarly tries "easy" tactics